src/components/visualizations/styles/_variables.scss

```diff
@@ -1,4 +1,4 @@
-// (C) 2007-2018 GoodData Corporation
+// (C) 2007-2020 GoodData Corporation
```

Comment:
Please check to revert the file in commit.
It could cause innecessary large number of files merge to develop branch later.